src/Shared/Converters/PortfolioHoldingsConverter.ts

```diff
@@ -0,0 +1,136 @@
+/* *
+ *
+ *  (c) 2009-2024 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Sophie Bremer
+ *  - Pawel Lysy
+ *  - Askel Eirik Johansson
+ *  - Jedrzej Ruta
+ *
+ * */
+
+
+'use strict';
+
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+
+import {
+    SecurityDetailsConverterOptions,
+    SecurityDetailsMetadata
+} from '../../SecurityDetails/SecurityDetailsOptions';
+import SecurityDetailsJSON from '../../SecurityDetails/SecurityDetailsJSON';
+import MorningstarConverter from '../MorningstarConverter';
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+
+export class PortfolioHoldingsConverter extends MorningstarConverter {
+
+
+    /* *
+     *
+     *  Constructor
+     *
+     * */
+
+
+    public constructor (
+        options?: SecurityDetailsConverterOptions
+    ) {
+        super(options);
+
+        this.metadata = {
+            columns: {},
+            ...(options && options.hasMultiple && { ids: [] }),
+            ...(options && options.hasMultiple && { isins: [] })
+        };
+    }
+
+
+    /* *
+     *
+     *  Properties
+     *
+     * */
+
+
+    public readonly metadata: SecurityDetailsMetadata;
+
+
+    /* *
+     *
+     *  Functions
+     *
+     * */
+
+
+    public override parse (
+        options: SecurityDetailsConverterOptions
+    ): void {
+        const metadata = this.metadata,
+            table = this.table,
+            userOptions = {
+                ...this.options,
+                ...options
+            },
+            security = userOptions.json as SecurityDetailsJSON.SecurityDetailsResponse,
+            hasMultiple = options.hasMultiple;
+
+
+        // Create table
+        const id = security.Id,
+            isin = security.Isin,
+            portfolioHoldings = security.Portfolios[0].PortfolioHoldings;
+
+        // Prepare table
+        table.deleteColumns();
```

Comment:
This line should be removed because it deletes the previous security in `SecurityCompareConnector`. Once removed everything should work both places.